pkg/xen-tools/Dockerfile

```diff
@@ -57,11 +59,11 @@ RUN rm -rf /out/usr/man
 RUN strip /out/usr/lib/* || :
 
 # When changing this version, also update SEABIOS_UPSTREAM_REVISION to whatever is in `Config.mk` in the xen source
-ENV XEN_VERSION 4.15.0
+ENV XEN_VERSION 4.19.0
 ENV XEN_SOURCE=https://downloads.xenproject.org/release/xen/${XEN_VERSION}/xen-${XEN_VERSION}.tar.gz
 ENV EXTRA_QEMUU_CONFIGURE_ARGS="--enable-libusb --enable-linux-aio \
-    --enable-vhost-net --enable-vhost-vsock --enable-vhost-scsi --enable-vhost-kernel \
-    --enable-vhost-user --enable-linux-io-uring --enable-zstd"
+--enable-vhost-net --enable-vhost-kernel \
+--enable-vhost-user --enable-linux-io-uring --enable-zstd"
```

Comment:
@shjala didn't we replace io-uring with other lib yet?

Reply:
no, we did some test with @jsfakian and it turned out that other mechanism are slow in comparison.

Reply:
@shjala, @rucoder it's a wrong PR, go to #4133 =)